README.rst

```diff
@@ -174,6 +173,62 @@ This will remove the container after every use so you will always have a fresh e
 
 You can also add an alias to your shell for "gallery-dl" or create a simple bash script and drop it somewhere in your $PATH to act as a shim for this command.
 
+Nix and Home Manager
+--------------------------
+
+For Nix users, there are a many of ways of using *gallery-dl*:
```

Comment:
This sounds kind of wrong. Why not just "there are many ways"?

Or we omit the first line entirely. This "guide" reads just fine without.